lib/logstash/inputs/file.rb

```diff
@@ -119,6 +135,17 @@ class LogStash::Inputs::File < LogStash::Inputs::Base
   # set the new line delimiter, defaults to "\n"
   config :delimiter, :validate => :string, :default => "\n"
 
+  # If this option is specified, when the file input discovers a file that
+  # was last modified before the specified timespan in seconds, the file is
+  # ignored. After it's discovery, if an ignored file is modified it is no
+  # longer ignored and any new data is read. The default is 24 hours.
+  config :ignore_older, :validate => :number, :default => 24 * 60 * 60
```

Comment:
@guyboertje I'm currently adding the close_older function also the filebeat. As we have now ignore_older and close_older, I suggest to set ignore_older to at least 72 hours or even "infinite" by default. The reasoning behind this is that we had some edge cases in the past. Any thoughts? https://github.com/elastic/filebeat/issues/181